README.md

```diff
@@ -1,37 +1,18 @@
-<h1 align="center">
-  <a href="">
-    <img src="/src/assets/music-releases.svg" alt="Project Banner Image">
-  </a>
-</h1>
 
 # Music Releases
+This week’s project in the Technigo boot camp was to build a website showing the latest album releases from a Spotify API, by writing JSX and using React components. 
 
-Replace this readme with your own information about your project.
+## The Problem
 
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+The real struggle with this project was to get a hang of how information is passed through the components using props. 
 
-## Getting Started with the Project
+At first I was very reluctant to make a separate component for the buttons and I was adding all the code for the buttons in the cover image component, but when I realised how much code the buttons needed I made a new component. And that was when it clicked for me why components are a great way of building code. They give you a really good overview of the code and it’s an easy way to pass data through the project. 
 
-### Dependency Installation & Startup Development Server
+If I would do it again I would make an individual css sheet for every component. I was afraid of having too many files in the beginning, as I found it very intimidating, and wasn’t sure on how to link them, but at the end I saw the point of having the css split up, and that’s when I made and individual	style sheet for the buttons. If I had more time I would continue structuring the CSS based on each component.
```

Comment:
I stayed with one CSS, and I agree, it got heavy! Let's practice that on our next project!